Makefile

```diff
@@ -9,6 +9,12 @@ else
 LDFLAGS ?= -ldflags=-X=github.com/aws/karpenter/pkg/utils/project.Version=$(shell git describe --tags --always)
 endif
 
+## Alias "gcut" to "cut" if running on Darwin
+UNAME := $(shell uname)
```

Comment:
why do you need this?  Seems like you're not using any gnu cut specific things?

Reply:
regular Darwin cut doesn't let you do `cut -d`